gdp-src-build/conf/templates/renesas-rcar-gen3.local.inc

```diff
@@ -0,0 +1,143 @@
+# Include generic GDP local configuration
+include local.inc
+
+# Add R-Car Gfx and MMP packages to install
+IMAGE_INSTALL_append_rcar-gen3 = "libdrm-kms kernel-module-gles gles-user-module libgbm-dev packagegroup-gstreamer1.0-plugins weston-bin kernel-module-pvrsrvkm"
+
+MACHINE ?= "m3ulcb"
+
+# This sets the SoC
+# H3: r8a7795, M3: r8a7796
+#SOC_FAMILY = "r8a7795"
+SOC_FAMILY ?= "r8a7796"
+
+# Add systemd configuration
+DISTRO_FEATURES_append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd"
```

Comment:
@chbae: I can't find a way to reply in line to your top comment about readme and the board setup name so I will add comments here.

Re r-car-m3: My comment assumes you are talking about the init.sh target parameter. I agree that "r-car-m3-starter-kit" is quite long but I found no good shortened form. Abbreviating "starter-kit" to 'sk' for example assumes knowledge about the board. The problem with "r-car-m3" is it only describes the SoC, not the board. We have R-Car M3 Starter Kit and R-Car M3 Salvator-X boards, as well as R-Car H3 SoC variants for example. So I don't think we can move to just r-car-m3.

Re README.md: Yes good catch. I thought about modifying it but I considered we needed to discuss whether this would be a supported board or not first. It could then be updated later, but I can update now if you prefer?